doc/source/formats/auxiliary.rst

```diff
@@ -217,3 +226,144 @@ To reload, pass the dictionary into :meth:`~MDAnalysis.coordinates.base.ProtoRea
     u2 = mda.Universe(PDB, TRR)
     for desc in descriptions:
         u2.trajectory.add_auxiliary(**desc)
+
+
+
+
+EDR Files
+---------
+EDR files are created by GROMACS during simulations and contain additional non-trajectory time-series data of the
+system, such as energies, temperature, or pressure. The EDRReader allows direct reading of these binary files
+and associating of the data with trajectory time steps just like the XVGReader does. As all functionality of the base AuxReaders (see XVGReader above) also works with
+the EDRReader, this section will highlight functionality unique to the EDRReader.
+
+
+Standalone Usage
+----------------
+The EDRReader is initialised by passing the path to an EDR file as an argument.
```

Comment:
Show an example (possibly commented out) with a normal path also rather than just a filename-handle from the testsuite as this will not be as familiar to people.

Reply:
Added, thank you!